package.json

```diff
@@ -44,7 +44,6 @@
         "lint:versions": "node ./scripts/lint-versions.js",
         "new-package": "cd projects/templates && plop",
         "postcustom-element-json": "node ./tasks/run-check-cem.js",
-        "postdocs:analyze": "node ./scripts/add-custom-properties.js --src=\"projects/documentation/custom-elements.json\"",
```

Comment:
add-custom-properties was meant to copy the token data from the vars and expressvars packages into the custom element manifest